<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmFiltrar</class>
 <widget class="QMainWindow" name="FrmFiltrar">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnSalir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../.png/salir2.png</normaloff>../.png/salir2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnReporte">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reporte</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../.png/agregar.png</normaloff>../.png/agregar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnReporteTotal">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>20</y>
      <width>125</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reporte total</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../.png/Reporte.png</normaloff>../.png/Reporte.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ELIJA UN NOMBRE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbEmpleado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
